js/client-chat.js

```diff
@@ -127,7 +127,12 @@
 			this.tabComplete.reset();
 			this.chatHistory.push(text);
 			text = this.parseCommand(text);
-			if (this.battle && this.battle.ignoreSpects && app.user.get('userid') !== this.battle.p1.id && app.user.get('userid') !== this.battle.p2.id) {
+			if (
+				this.battle && this.battle.ignoreSpects &&
+				app.user.get('userid') !== this.battle.p1.id &&
+				app.user.get('userid') !== this.battle.p2.id &&
+				!(text.startsWith('/') || text.startsWith('!'))
```

Comment:
Broadcast commands should still be unusable, as should `/me`.

```suggestion
				!(text.startsWith('/') && !text.startsWith('/me'))
```